Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN apk update && \
     && apk del .ruby-builddeps \
     && rm -rf /var/cache/apk/*
 
-RUN adduser -g msfconsole -D $MSF_USER
+RUN adduser -g msfconsole -D $MSF_USER -u 1000
```

Comment:
Does:

RUN adduser -g msfconsole -D $MSF_USER -u `id -u`

have the desired effect?

Reply:
Trying it now...

Reply:
sadly not. the image is built on docker hub and the id is baked into the image :/